src/localize/languages/pt-PT.json

```
@@ -0,0 +1,470 @@
+            "popup-top-right": "Pop-up no canto superior direito"
+          }
+        }
+      },
+      "layout": {
+        "fit": "Fit",
+        "fits": {
+          "contain": "Conter",
+          "cover": "Tapar",
+          "fill": "Preencher"
+        },
+        "position": {
+          "x": "Percentagem da localização horizontal",
+          "y": "Percentagem da localização vertical"
+        }
+      },
+      "media_action_conditions": {
+        "all": "Todas as oportunidades",
+        "hidden": "Ao ocultar o navegador/aba",
+        "never": "Nunca",
+        "selected": "Ao selecionar",
+        "unselected": "Ao desselecionar",
+        "visible": "Ao mostrar o navegador/aba"
+      },
+      "timeline": {
+        "clustering_threshold": "A contagem de eventos nos quais eles são agrupados (0 = sem agrupamento)",
+        "media": "A mídia que a linha do tempo exibe",
+        "medias": {
+          "all": "Todos os tipos de mídia",
+          "clips": "Clipes",
+          "snapshots": "Instantâneos"
+        },
+        "show_recordings": "Mostrar gravações",
+        "window_seconds": "A duração padrão da visualização da linha do tempo em segundos"
+      }
+    },
+    "dimensions": {
+      "aspect_ratio": "Proporção padrão (e.g. '16:9')",
+      "aspect_ratio_mode": "Modo de proporção",
+      "aspect_ratio_modes": {
+        "dynamic": "A proporção se ajusta à mídia",
+        "static": "Proporção estática",
+        "unconstrained": "Proporção irrestrita"
+      }
+    },
+    "image": {
+      "layout": "Layout",
+      "mode": "Modo de visualização de imagem",
+      "modes": {
+        "camera": "Instantâneo da câmera do Home Assistant, da entidade de câmera",
+        "screensaver": "Logo Frigate embutido",
+        "url": "Imagem arbitrária especificada por URL"
+      },
+      "refresh_seconds": "Número de segundos após o qual atualizar (0 = nunca)",
+      "url": "Imagem arbitrária especificada por URL"
+    },
+    "live": {
+      "auto_mute": "Silenciar câmeras ao vivo automaticamente",
+      "auto_pause": "Pausar câmeras ao vivo automaticamente",
+      "auto_play": "Reproduzir câmeras ao vivo automaticamente",
+      "auto_unmute": "Ativar automaticamente o som das câmeras ao vivo",
+      "controls": {
+        "editor_label": "Controles da visualização ao vivo"
+      },
+      "draggable": "A visualização ao vivo das câmeras pode ser arrastada/deslizada",
+      "layout": "layout",
+      "lazy_load": "As câmeras ao vivo são carregadas lentamente",
+      "lazy_unload": "As câmeras ao vivo são descarregadas preguiçosamente",
+      "preload": "Pré-carregar a visualização ao vivo em segundo plano",
+      "show_image_during_load": "Mostar imagem durante o carregamento",
+      "transition_effect": "Efeito de transição de câmera ao vivo"
+    },
+    "media_viewer": {
+      "auto_mute": "Silenciar mídia automaticamente",
+      "auto_pause": "Pausar mídia automaticamente",
```

Brazilian expression not used in pt_PT
```suggestion
      "auto_pause": "Parar mídia automaticamente",
```